lib/main.dart

```diff
@@ -9,7 +9,7 @@ Future main() async {
   final environment =
       enumFromString(Environments.values, const String.fromEnvironment('ENV')) ??
           Environments.development;
-  await dotenv.load(fileName: '${environment.toString().split('.').last}.env');
+  await dotenv.load(fileName: 'assets/${environment.toString().split('.').last}.env');
```

Comment:
what about adding them into a folder under assets like environments? since probably we would have others for images and so on also I think its a little bit weird to get the environment split and add .env what about using some extensions here something like this:
```
extension EnviromentPath on Environments {
  String get fileName {
    switch (this) {
      case Environments.development:
        return "development.env";
      case Environments.staging:
        return "staging.env";
      case Environments.production:
        return "production.env";
    }
  }

  String get path => "assest/enviroments/" + fileName;
}
```